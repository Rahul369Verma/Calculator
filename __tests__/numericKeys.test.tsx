import React from "react";
const renderer = require("react-test-renderer");
import NumberKeys from "../components/numericKeys";

it("renders NumericKeys correctly", () => {
  const tree = renderer.create(<NumberKeys handleClick={jest.fn()} />);
  expect(tree).toMatchSnapshot();
});
