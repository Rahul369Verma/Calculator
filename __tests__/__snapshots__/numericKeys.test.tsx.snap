// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders NumericKeys correctly 1`] = `
<div
  className="numbers"
>
  <button
    className="btn-cal btn-primary"
    data-testid="seven"
    onClick={[Function]}
    type="button"
  >
    7
  </button>
  <button
    className="btn-cal btn-primary"
    data-testid="eight"
    onClick={[Function]}
    type="button"
  >
    8
  </button>
  <button
    className="btn-cal btn-primary"
    data-testid="nine"
    onClick={[Function]}
    type="button"
  >
    9
  </button>
  <button
    className="btn-cal btn-primary"
    data-testid="four"
    onClick={[Function]}
    type="button"
  >
    4
  </button>
  <button
    className="btn-cal btn-primary"
    data-testid="five"
    onClick={[Function]}
    type="button"
  >
    5
  </button>
  <button
    className="btn-cal btn-primary"
    data-testid="six"
    onClick={[Function]}
    type="button"
  >
    6
  </button>
  <button
    className="btn-cal btn-primary"
    data-testid="one"
    onClick={[Function]}
    type="button"
  >
    1
  </button>
  <button
    className="btn-cal btn-primary"
    data-testid="two"
    onClick={[Function]}
    type="button"
  >
    2
  </button>
  <button
    className="btn-cal btn-primary"
    data-testid="three"
    onClick={[Function]}
    type="button"
  >
    3
  </button>
  <button
    className="btn-cal btn-primary col-span-2"
    data-testid="zero"
    onClick={[Function]}
    type="button"
  >
    0
  </button>
  <button
    className="btn-cal btn-primary"
    data-testid="full-stop"
    onClick={[Function]}
    type="button"
  >
    .
  </button>
</div>
`;
